javatests/arcs/core/data/proto/SchemaProtoTest.kt

```diff
@@ -173,4 +181,89 @@ class SchemaProtoDecoderTest {
             )
         )
     }
+
+    @Test
+    fun encode_schema() {
+        val schema = Schema(
+            names = setOf(SchemaName("a"), SchemaName("b")),
+            fields = SchemaFields(
+                singletons = mapOf("txt" to FieldType.Text),
+                collections = mapOf("nums" to FieldType.Number)
+            ),
+            hash = "myHash"
+        )
+
+        assertThat(schema.encode()).isEqualTo(
+            SchemaProto.newBuilder()
```

Comment:
This is fine, but was wondering why you were not using the text proto parser.

Reply:
I prefer not using it, because you can't find usages of proto fields in IDEs/CS